packages/webpack/preset.js

```diff
@@ -20,4 +20,14 @@ module.exports = {
     join(__dirname, 'mixins', 'start'),
     join(__dirname, 'mixins', 'stats'),
   ],
+  mixinTypes: {
+    browser: {
+      mainFiles: ['mixin.browser', 'mixin.runtime', 'mixin'],
+      mainFields: ['mixin:browser', 'mixin:runtime', 'mixin'],
```

Comment:
How about simplifying the config to something like this?
```
  mixinTypes: {
    browser: ['browser', 'runtime'],
    server: ['server', 'runtime'],
  },
```

Reply:
I thought about that, too, but ultimately decided to go for the full-on `enhanced-resolve` config here for two reasons:

- more flexibility for downstream (e.g. support for *.ts mixins)
- less indirection/magic (we just use what is being passed to us w/o any assumptions)